src/config/dynamic-theme-fixes.config

```
@@ -115,6 +115,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.tvland.com
```

```suggestion
tvland.com
```

Also re-order as appriopate.